### DISCUSSION TITLE
Decentralized stable coin for gold.

### DISCUSSION THREAD

**Discussion by silicology1** [AcalaNetwork/Acala#2645] [1. Ideas]
The USD is issued by the US government. While they aim to maintain only a 10-15% inflation rate per year, its predictability is under centralized control. Throughout history, there have been instances of bank crises leading to a sudden devaluation of the USD, and history may repeat itself.

Using the USD as the mainstream cryptocurrency contradicts the ethos of cryptocurrency, as it should be decentralized. Its use needs to be limited and avoided from dominating the market.  Furthermore, since the USD is a national currency primarily serving the needs of the United States, it is less likely to effectively meet the diverse economic requirements of international nations. For example, if you belong to some other country and hold USD, the 10% USD inflation will transfer wealth from you to US. 

While the USD may be appropriate for short-term use, like when you want to use the money within a year, gold is perceived as a superior option for long-term savings and is better at providing a hedge against inflation or crypto-speculation.

Though there are few gold backed cryptocurrency

    Tether Gold (XAUT)
    DigixGlobal (DGX)
    PAX Gold (PAXG)
    Gold Coin (GLC)
    Perth Mint Gold Token (PMGT)
    Meld
But they are not fully decentralized like Acala. Gold pegged decentralized stable coin can help to fill the gap. It can be great if Acala provide such service. 

![value_during_crisis](https://github.com/AcalaNetwork/Acala/assets/2051004/651b50c9-1a29-4423-8015-10b9758efdb0)

**Comment by xlc** [comment]
Thanks for the suggestion. I can see this could be helpful, but there is an important question to be answered: how much demand do we expect for such product?

**Reply by silicology1** [reply]
Similar demand as other gold stable coin, also cryptodevelopers and cryptocommunity needs to understand the problem of USD, specially people from other countries and use Gold stable coin often rather than USD.
